src/components/Reports/ReportsPageRanking/ReportsPageRanking.js

```diff
@@ -69,23 +67,33 @@ class ReportsPageRanking extends React.Component {
     return (
       <>
         {pages === null ? (
-          <Loading />
+          <div className="loading-box" />
         ) : (
-          <div className="page-ranking--container">
-            <header>
-              <FormattedMessage id="reports_pageranking.top_pages" />
-            </header>
-            {pages.map((item, index) => (
-              <div key={index} className="page-ranking--item">
-                <strong>{index + 1}</strong>
-                <p className="page-name">{item.name}</p>
-                <p className="visits">
-                  <strong>
-                    {item.totalVisits} <FormattedMessage id="reports_pageranking.total_visits" />
-                  </strong>
-                </p>
-              </div>
-            ))}
+          <div className="wrapper-ranking">
+            <div className="reports-box">
+              <small className="title-ranking">
+                <FormattedMessage id="reports_pageranking.top_pages" />
+              </small>
+              {pages.map((item, index) => (
+                <div key={index} className="page-ranking--item">
+                  <p className="text-ranking">
+                    <strong>{index + 1}</strong>
+                  </p>
+                  <a
+                    className="link-ranking"
+                    href="{item.name}"
```

Comment:
El código correcto es así, como te mostré en [mi comentario anterior](https://github.com/FromDoppler/doppler-webapp/pull/93#discussion_r269709062):

```suggestion
                    href={item.name}
```

En caso contrario lo toma como texto: 

![image](https://user-images.githubusercontent.com/1157864/55156239-29093b00-5138-11e9-89ea-f3de3087b519.png)